### PR TITLE
Fix 54 comment nodes

### PR DESCRIPTION
**Pull request by jpnelson**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -701,7 +711,10 @@ const members = {
     get () {
       let textContent = '';
       eachChildNode(this, function (node) {
-        textContent += node.textContent;
+        const isCommentNode = node.nodeType === 8;
```
Comment (treshugart) [inline src/index.js]: Instead of creating a separate variable, you can probably get the same point across using the `Node.COMMENT_NODE` constant.

Comment (jpnelson) [inline src/index.js]: sure, I didn't know about that

File: src/index.js [hunk 2]
```diff
@@ -533,7 +534,16 @@ const members = {
     get () {
       let innerHTML = '';
       eachChildNode(this, function (node) {
-        innerHTML += node.nodeType === 1 ? node.outerHTML : getEscapedTextContent(node);
+        const getHtmlNodeOuterHtml = (node) => node.outerHTML;
+        const getOuterHtmlByNodeType = {
```
Comment (treshugart) [inline src/index.js]: Probably good to move these constants outside of the loop.

**Comment by lukebatchelor** [comment]
LGTM (y)

**Pull request merged**